### PR TITLE
[new] Add lightSlider with git autoupdate. close #5594.

### LINKED ISSUE #5594: [Request] Add lightslider
lightSlider
https://github.com/sachinchoolur/lightslider
MIT
http://sachinchoolur.github.io/lightslider/

### PR DESCRIPTION
**Pull request by Piicksarn**
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct 

**`Git repository :`** https://github.com/sachinchoolur/lightslider/tree/1.1.3
**`Star :  655  Fork :  249   Watch :  51`**

@Amomo, would you please have a look at the PR?
Thank you!

### CODE REVIEW

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look? Thank you~

File: ajax/libs/lightslider/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "name": "Sachin N",
+    "email": "sachi77n@gmail.com",
+    "url": "https://github.com/sachinchoolur"
+  },
+  "homepage": "https://github.com/sachinchoolur/lightslider",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sachinchoolur/lightslider.git"
+  },
+  "license": {
```
Comment (PeterDaveHello) [inline ajax/libs/lightslider/package.json]: https://github.com/cdnjs/cdnjs/issues/5543

**Comment by Piicksarn** [comment]
@Amomo, would you mind to take a look at the pr again?
Thank you! :smiley:

**Comment by Amomo** [comment]
@PeterDaveHello I think Piicksarn has corrected the license field.
Could you take a look again? thank you!

**Pull request merged**